docs/source/rule-testing.rst

```diff
@@ -76,6 +76,84 @@ For example, to replace a time based field with ``last_hour``:
   }
 
 
+Validating Log Schemas
+~~~~~~~~~~~~~~~~~~~~~~
+
+In some cases, there may be logs coming into StreamAlert that are of a known type, but you do not have a current need to apply rule
```

Comment:
`may be incoming logs to StreamAlert with a known type but without specific rules`